docs/tutorials/demo/Alchemiscale Demo.ipynb

```diff
@@ -0,0 +1,2774 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6d27b499-55a1-4050-8c69-826a130707bd",
+   "metadata": {},
+   "source": [
+    "# Demo - Using `alchemiscale` to evaluate a relative binding free energy network"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e3ea5f9b-97ee-4bc2-bfd7-c5cde5e0420c",
+   "metadata": {},
+   "source": [
+    "This notebook details the process of running a relative binding free energy calculation using the [openfe](https://github.com/OpenFreeEnergy/openfe/) toolkit and the execution platform [alchemiscale](https://github.com/openforcefield/alchemiscale).\n",
+    "\n",
+    "The notebook is broken down into the following section:\n",
+    "  1. Creating an `AlchemicalNetwork`\n",
+    "  2. Submitting an `Alchemicalnetwork` to `alchemiscale`\n",
+    "  3. Monitoring the progress of an `alchemiscale` submission\n",
+    "  4. Gathering results and analyzing them using `cinnabar`\n",
+    "\n",
+    "Credits:\n",
+    "- Irfan Alibay ([@ialibay](https://github.com/IAlibay)) for initial structure and content\n",
+    "- David Dotson (@dotsdl) for expansion and editing for use as a tutorial"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "177075c1-0bf3-4f59-88fd-2b2122b34555",
+   "metadata": {},
+   "source": [
+    "## Creating an `AlchemicalNetwork` for the `tyk2` benchmark system"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8d1d6468-70a5-4d0b-b4bf-8d7093b6be9a",
+   "metadata": {},
+   "source": [
+    "We first illustrate how to build `AlchemicalNetwork` objects,\n",
+    "from a starting point of chemical models stored in sdf and pdb files.\n",
+    "\n",
+    "An `AlchemicalNetwork` is used to represent an entire network of calculations,\n",
+    "and is composed of many smaller objects:\n",
+    "\n",
+    "- An `AlchemicalNetwork` composed of \n",
+    "  - each node a `ChemicalSystem`\n",
+    "    - each containing many components, such as `SmallMoleculeComponent`, `ProteinComponent`\n",
```

Comment:
Would it make sense to have these link to the original docs? https://docs.openfree.energy/en/stable/reference/api/systems_and_components.html etc..

Reply:
Not sure how stable those specific page links will be, so would rather not here. We do link to the `openfe` repo at the top of the notebook.